temporal/t.remove/t.remove.py

```diff
@@ -117,16 +116,28 @@ def main():
             dataset_list.append(dataset_name)
 
     statement = ""
-
     # Create the pygrass Module object for g.remove
     remove = pyg.Module("g.remove", quiet=True, flags="f", run_=False)
 
+    if not force and not clean:
+        grass.message(_("The following data base element files would be deleted:"))
```

Comment:
```suggestion
        grass.message(_("The following data base element files will be deleted:"))
```

Reply:
but the will not removed, but only with force they will be removed

Reply:
Yeah, it's just that would doesn't sound good to me. Besides, after the list of maps, there's the instruction saying, add the -f flag to actually remove, no? Indeed, I'd change the "would" also in g.remove... Not so important anyway